### DISCUSSION TITLE
Welcome to PenguinMod GitHub Forums

### DISCUSSION THREAD

**Discussion by SharkPool-SP** [PenguinMod/PenguinMod-Home#76] [Announcements]
### Hi, What’s This?
Use this place to discuss **PenguinMod**, get questions answered, and more! Don’t bother opening new issues :P

**Comment by Gen1x-ALT** [comment]
the j

**Comment by SpareEnderboy** [comment]
#75 made when i didnt know this existed

**Comment by mariocraft987** [comment]
maybe more tips?

**Comment by RedMan13** [comment]
> ### Hi, What’s This?
> 
> Use this place to discuss **PenguinMod**, get questions answered, and more! Don’t bother opening new issues :P

nuh uh site issues going the issues

**Comment by RedMan13** [comment]
also hello chat :bleh

**Reply by mariocraft987** [reply]
Yo

**Comment by RedMan13** [comment]
nvm ignore me this is the. correct place to forym
